src/black/__init__.py

```diff
@@ -1132,8 +1132,12 @@ def get_features_used(node: Node) -> Set[Feature]:  # noqa: C901
                 features.add(Feature.F_STRINGS)
 
         elif n.type == token.NUMBER:
-            if "_" in n.value:  # type: ignore
+            assert isinstance(n, Leaf)
+            if "_" in n.value:
                 features.add(Feature.NUMERIC_UNDERSCORES)
+            elif n.value.endswith("L"):
```

Comment:
```suggestion
            elif n.value.endswith(("l", "L")):
```
It can be lowercase too.

Also, we could look for `010` (old octal syntax), which doesn't work in 3.0 any more. This is in https://docs.python.org/3/whatsnew/3.0.html#integers and PEP 3127. We could check that the first char is 0 and the second is a digit.

Reply:
great catch, thank you!